pkg/converter/convert_unix.go

```diff
@@ -450,6 +450,23 @@ func Unpack(ctx context.Context, ra content.ReaderAt, dest io.Writer, opt Unpack
 	return nil
 }
 
+// IsNydusBlob returns true when the specified digest of content exists in
+// the content store and it's nydus blob format.
+func IsNydusBlob(ctx context.Context, cs content.Store, desc ocispec.Descriptor) bool {
+	if desc.Annotations == nil {
+		return false
+	}
+
+	_, err := cs.Info(ctx, desc.Digest)
+	if err != nil {
+		return false
+	}
+
+	hasMediaType := desc.MediaType == MediaTypeNydusBlob
+	_, hasAnno := desc.Annotations[LayerAnnotationNydusBlob]
```

Comment:
Why is only nydus blob media type not enough to judge it's a nydus blob layer?

Reply:
Maybe we could just check by media type, but recently, we won't assume that the media type must be `"application/vnd.oci.image.layer.nydus.blob.v1"`, maybe a tgz media type can be supported in future? So we should always trust the annotation.

Reply:
Then how about not checking media type but just checking annotation? Otherwise, this method malfunctions when "application/vnd.oci.image.layer.nydus.blob.v1" is abandoned.

Reply:
Okay, it makes sense, fixed.